test/lib/faraday_bang/bang_test.rb

```diff
@@ -8,7 +8,14 @@
 
   [Faraday, Faraday.new].each do |klass|
 
-    Faraday::Connection::METHODS.each do |verb|
+    Faraday::Bang::SUPPORTED_HTTP_METHODS.each do |verb|
```

Comment:
You're only testing supported methods, what about the separate `options!` method? There needs to be a test for that.